frontend/src/app/offers/offer-detail/offer-detail.component.html

```diff
@@ -48,14 +48,26 @@ <h3 class="card-header">Zaangażowanie czasowe</h3>
         </div>
         <div class="col-xs-12 col-sm-6 col-md-4">  
             <div class="card text-center">
-                <div class="alert alert-danger">
-                    <h2>Możesz pomóc?</h2><br>
-                    Twoja pomoc jest ważna.<br>
-                    <b><h3>Potrzebujemy Ciebie!</h3></b><br>
-                    <a class="btn btn-light" href="{{ getJoinViewUrl(offer) }}">Zgłoś się na ten wolontariat</a>
-                </div>
-                <h3 class="card-header">Rekrutacja trwa do: {{ offer.recruitmentEndDate | date:'shortDate' }}</h3>
+                <div *ngIf="!offer.joined; else questionsBlock">
```

Comment:
I don't know if `questionsBlock` is a good name here, what do You think about `alreadyJoinedBlock`?

Reply:
Will do